### PR TITLE
SDK configuration

### PR DESCRIPTION
**Pull request by compnerd**
Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves [SR-NNNN](https://bugs.swift.org/browse/SR-NNNN).

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test windows platform

**Comment by compnerd** [comment]
CC: @drodriguez

File: stdlib/public/Platform/CMakeLists.txt [hunk 1]
```diff
@@ -67,34 +67,6 @@ foreach(sdk ${SWIFT_SDKS})
...
-    endif()
-
-    set(GLIBC_INCLUDE_PATH "${GLIBC_SYSROOT_RELATIVE_INCLUDE_PATH}")
-    set(GLIBC_ARCH_INCLUDE_PATH "${GLIBC_SYSROOT_RELATIVE_ARCH_INCLUDE_PATH}")
-
-    if(NOT "${SWIFT_SDK_${sdk}_ARCH_${arch}_PATH}" STREQUAL "/" AND NOT "${sdk}" STREQUAL "ANDROID")
-      set(GLIBC_INCLUDE_PATH "${SWIFT_SDK_${sdk}_ARCH_${arch}_PATH}${GLIBC_INCLUDE_PATH}")
-      set(GLIBC_ARCH_INCLUDE_PATH "${SWIFT_SDK_${sdk}_ARCH_${arch}_PATH}${GLIBC_ARCH_INCLUDE_PATH}")
-    endif()
-
```
Comment (drodriguez) [inline stdlib/public/Platform/CMakeLists.txt]: I think if you remove this last part, `GLIBC_INCLUDE_PATH` and `GLIBC_ARCH_INCLUDE_PATH` will not be set, and the generation of the module map for building will not work.

I think at least the last two blocks have to be recovered, the set the variables used in one of the generations, and deal with having sysroots in different paths as the final installation point.

**Comment by drodriguez** [comment]
I don’t have an opinion on the reordering of the summary messages, but maybe some tooling is depending on that output. Somebody else should accept or reject this with proper knowledge.

File: stdlib/public/Platform/CMakeLists.txt (hunk 1 above)
Comment (compnerd) [reply stdlib/public/Platform/CMakeLists.txt]: The point of that particular commit is to _remove_ this entire block.  We now provide proper mechanisms via the cache to control these.  These variables are no longer used.

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
@swift-ci please test

**Review by drodriguez — APPROVED** [review]
I like the clean up of moving the paths to one place, which is not hidden.

If your OCD can deal with the double slash at the start of all module map header paths, then I'm fine with it (I think the Android check there was not really necessary).

Finally, if you really thing the `${CMAKE_LIBRARY_ARCHITECTURE}` is not really necessary for any Linux distribution, then go ahead.

**Comment by compnerd** [comment]
> I like the clean up of moving the paths to one place, which is not hidden.
> 
> If your OCD can deal with the double slash at the start of all module map header paths, then I'm fine with it (I think the Android check there was not really necessary).

LMAO, triggered!  Yes, it is bothering me, and I intend to clean that up subsequently.  I wanted to get this working first :-).

> Finally, if you really thing the `${CMAKE_LIBRARY_ARCHITECTURE}` is not really necessary for any Linux distribution, then go ahead.

**Pull request merged**